kotlin.html.markdown

```diff
@@ -261,6 +261,14 @@ fun helloWorld(val name : String) {
         ctr++
     } while (ctr < 10)
 
+    /*
+    "if" can be used as a function that returns a value.
```

Comment:
Indeed you're right, but then I suggest the same change for the "when" keyword.. Even if it resembles a function, it's more correct to call it an expression too.
What do you think?

> Il giorno 12 set 2016, alle ore 14:14, ven <notifications@github.com> ha scritto:
> 
> In kotlin.html.markdown:
> 
> > @@ -261,6 +261,14 @@ fun helloWorld(val name : String) {
> >          ctr++
> >      } while (ctr < 10)
> >  
> > +    /*
> > +    "if" can be used as a function that returns a value.
> can be used as an expression
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>